### PR TITLE
Run database migration job after installation/upgrade

### PR DESCRIPTION
**Pull request by sliverc**
For an upgrade to complete database migrations need to be run. This is currently done during post phase but once helm implements stages it should be moved into pre phase.

### CODE REVIEW

File: templates/post-install-job-dbmigrate.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  # TODO: this should be done during pre phase but to do so need
```
Comment (tongpu) [inline templates/post-install-job-dbmigrate.yaml]: There is a word missing somewhere in that sentence.

File: README.md [hunk 2]
```diff
@@ -7,15 +7,15 @@ This chart bootstraps a Timed application deployment on a [Kubernetes](https://k
 ## Prerequisites
 
 - Kubernetes 1.7+ with Beta APIs enabled
-- PV provisioner support in the underlying infrastructure
```
Comment (tongpu) [inline README.md]: Why was this removed? Is dynamic PV provisioning not required anymore?

Comment (tongpu) [reply README.md]: Ok, technically it's not required, but as the database chart will need it in the future we should keep it.

Comment (sliverc) [reply README.md]: Agree. Added again.

File: templates/post-install-job-dbmigrate.yaml (hunk 1 above)
Comment (sliverc) [reply templates/post-install-job-dbmigrate.yaml]: Clarified doc string.

**Comment by sliverc** [comment]
@tongpu ready for review again

**Review by tongpu — APPROVED** [review]
Thanks for the changes, looks good to me!
